Source/WebCore/css/CSSSelector.h

```diff
@@ -116,25 +116,25 @@ struct PossiblyQuotedIdentifier {
             NthLastOfType,
             Link,
             Visited,
-            Any,
+            WebKitAny,
             AnyLink,
-            AnyLinkDeprecated,
+            WebKitAnyLink,
             Autofill,
-            AutofillAndObscured,
-            AutofillStrongPassword,
-            AutofillStrongPasswordViewable,
+            WebKitAutofillAndObscured,
+            WebKitAutofillStrongPassword,
+            WebKitAutofillStrongPasswordViewable,
             Hover,
-            Drag,
+            WebKitDrag,
             Focus,
             FocusVisible,
             FocusWithin,
             Active,
             Checked,
             Enabled,
-            FullPageMedia,
+            WebKitFullPageMedia,
             Default,
             Disabled,
-            HtmlDocument, // for internal use only with forms in table case in UA stylesheet
+            InternalHtmlDocument, // for internal use in html.css
```

Comment:
Given we are spelling `-webkit-` as `WebKit`, could we make this `InternalHTMLDocument`?

Reply:
Sure, I special cased it